lib/src/WebSocketConnectionImpl.cc

```diff
@@ -159,7 +159,7 @@ void WebSocketConnectionImpl::sendWsData(const char *msg,
     tcpConnectionPtr_->send(std::move(bytesFormatted));
 }
 
-void WebSocketConnectionImpl::send(const std::string &msg,
+void WebSocketConnectionImpl::send(std::string_view msg,
```

Comment:
```suggestion
void WebSocketConnectionImpl::send(const std::string_view msg,
```

This can be const-qualified.

Reply:
It doesn't make a difference though.
```c++
void send(const char* const data, const size_t len);
```
It only disallows the reassignment of the string_view from pointing to somewhere else by us, which we don't do within the function definition.

Reply:
Yes, it only prevents reassignment, which is definitely better than no-const. I think it is always recommended to use const as a best practice.